lib/entries.js

```diff
@@ -96,77 +92,78 @@ function storage(name, storage, pushover) {
 
   // store new documents using the storage mechanism
   function create (docs, fn) {
-      with_collection(function(err, collection) {
-        if (err) { fn(err); return; }
-        // potentially a batch insert
-        var firstErr = null,
-            totalCreated = 0;
-
-        docs.forEach(function(doc) {
-            collection.update(doc, doc, {upsert: true}, function (err, created) {
-                firstErr = firstErr || err;
-                totalCreated += created;
```

Comment:
This is especially sketchy, as `created` is an object, but the initial value of `totalCreated` is a number, turning `totalCreated` into a string of the form `0{"ok":1,"nModified":1,"n":1}{"ok":1,"nModified":1,"n":1}`...